frontend/packages/enduser-frontend/src/store/modules/applications.ts

```diff
@@ -42,6 +42,7 @@ const module: Module<ApplicationsState, RootState> = {
       commit(types.CLEAR_FORM, 'application')
       commit(types.CLEAR_DAYCARE_FORM)
       const application = await applicationApi.getApplication(applicationId)
+      console.log(application)
```

Comment:
Debugging leftovers?

Reply:
😳

Reply:
I accidentally committed code that I wasn't supposed to 😅